nimx/private/text_drawing.nim

```diff
@@ -124,6 +124,10 @@ void main() {
 }
 """,
 """
+#ifdef GL_ES
+#extension GL_OES_standard_derivatives : enable
+precision mediump float;
+#endif
```

Comment:
I think these changes are not needed because you handle the headers in `composition.nim`